config/locales/models/report.en.yml

```diff
@@ -13,7 +13,6 @@ en:
     header:
       report:
         financial_quarter: Financial quarter
-        state: State
```

Comment:
I wouldn't necessarily infer the change was intentional, especially if it wouldn't have worked, btw.